__tests__/app.test.js

```diff
@@ -5,9 +5,24 @@ const app = require("../app");
 const seed = require("../db/seeds/seed");
 
 afterAll(() => db.end());
-/* beforeEach(() => seed(data)); */
+beforeEach(() => seed(data));
 
 describe("GET /api/", () => {
+    test("200:status code and responds with object describing all the available endpoints", async () => {
+        const { status, body } = await request(app).get("/api/");
+        const { endpoints } = body;
+        const endpointKeys = Object.keys(endpoints);
+
+        endpointKeys.forEach((key) => {
+
+            const actualKeys = Object.keys(endpoints[key]);
+            const expectedKeys = ["description", "queries", "requestBody", "exampleResponse"];
+            expect(actualKeys).toEqual(expectedKeys);
+        });
+        expect(status).toBe(200);
+
```

Comment:
You are checking that the keys are there but you aren't checking the values. I'd consider requiring in the JSON at the top 😄